lib/core/output_sink/console_sink.rb

```diff
@@ -0,0 +1,45 @@
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+#
+
+# frozen_string_literal: true
+
+require 'core/output_sink'
+require 'utility/logger'
+
+module Core::OutputSink
+  class ElasticSink < Core::OutputSink::BaseSink
+    def ingest(_document)
+      print_header 'Got a single document:'
+    end
```

Comment:
```suggestion
    def ingest(document)
      print_header 'Got a single document:'
      Utility::Logger.info document
    end
```

To be consistent with `ingest_multiple(documents)

Reply:
Good call, fixed!